src/mono/browser/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -147,10 +147,23 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                     }
                 case "Debugger.scriptParsed":
                     {
-                        if (args["url"]?.ToString()?.Contains("/_framework/") == true) //is from dotnet runtime framework
+                        if (!Contexts.TryGetCurrentExecutionContextValue(sessionId, out ExecutionContext context))
+                            return false;
+                        try
                         {
-                            if (Contexts.TryGetCurrentExecutionContextValue(sessionId, out ExecutionContext context))
+                            var callStack = args["stackTrace"]?["callFrames"]?.Value<JArray>();
+                            var callStackFunctioName = callStack?.Count > 0 ? callStack?[0]?["functionName"]?.Value<string>() : null;
```

Comment:
```suggestion
                            var callStackFunctionName = callStack?.Count > 0 ? callStack?[0]?["functionName"]?.Value<string>() : null;
```

Reply:
or maybe rename to `topFrameFunctionName`

Reply:
done.